### PR TITLE
Remove extra fields from Manage organisation details (PTCD-559)

### PR DESCRIPTION
**Pull request by gunndabad**

### CODE REVIEW

File: src/Dfc.CourseDirectory.Core/Validation/ProviderValidation/RuleBuilderExtensions.cs [hunk 1]
```diff
@@ -22,9 +22,9 @@ public static void MarketingInformation<T>(this IRuleBuilderInitial<T, string> f
         {
             field
                 .NotEmpty()
-                    .WithMessage("Enter provider marketing information")
+                    .WithMessage("Enter brief overview of your organisation for employers")
```
Comment (jameslawesynergy) [inline src/Dfc.CourseDirectory.Core/Validation/ProviderValidation/RuleBuilderExtensions.cs:25]: "Enter **a** brief..."?

File: src/Dfc.CourseDirectory.WebV2/Features/Providers/EditProviderInfo.cs [hunk 2]
```diff
@@ -58,52 +48,41 @@ public class Handler :
             _cosmosDbQueryDispatcher = cosmosDbQueryDispatcher;
             _currentUserProvider = currentUserProvider;
             _clock = clock;
-            _validator = new CommandValidator(CourseDirectoryNameIsEditable, MarketingInformationIsEditable);
         }
 
-        private bool CourseDirectoryNameIsEditable =>
-            AuthorizationRules.CanUpdateProviderCourseDirectoryName(_currentUserProvider.GetCurrentUser());
+        ProviderType IRestrictProviderType<Query>.ProviderType => ProviderType.Apprenticeships;
```
Comment (jameslawesynergy) [inline src/Dfc.CourseDirectory.WebV2/Features/Providers/EditProviderInfo.cs:53]: Declare as `private` memeber?

File: src/Dfc.CourseDirectory.WebV2/Features/Providers/EditProviderInfo.cs [hunk 3]
```diff
@@ -58,52 +48,41 @@ public class Handler :
...
 
-        private bool MarketingInformationIsEditable =>
-            AuthorizationRules.CanUpdateProviderMarketingInformation(_currentUserProvider.GetCurrentUser());
+        ProviderType IRestrictProviderType<Command>.ProviderType => ProviderType.Apprenticeships;
```
Comment (jameslawesynergy) [inline src/Dfc.CourseDirectory.WebV2/Features/Providers/EditProviderInfo.cs:55]: Declare as `private` memeber?

File: src/Dfc.CourseDirectory.WebV2/Features/Providers/EditProviderInfo.cs [hunk 4]
```diff
@@ -112,56 +91,28 @@ public class Handler :
...
             if (result == null)
             {
-                throw new ErrorException<ProviderDoesNotExist>(new ProviderDoesNotExist());
+                throw new ResourceDoesNotExistException(ResourceType.Provider, providerId);
             }
 
             return result;
         }
 
+        Guid IRestrictProviderType<Query>.GetProviderId(Query request) => request.ProviderId;
```
Comment (jameslawesynergy) [inline src/Dfc.CourseDirectory.WebV2/Features/Providers/EditProviderInfo.cs:107]: Declare as `private` memeber?

File: src/Dfc.CourseDirectory.WebV2/Features/Providers/EditProviderInfo.cs [hunk 5]
```diff
@@ -112,56 +91,28 @@ public class Handler :
...
+
+        Guid IRestrictProviderType<Command>.GetProviderId(Command request) => request.ProviderId;
```
Comment (jameslawesynergy) [inline src/Dfc.CourseDirectory.WebV2/Features/Providers/EditProviderInfo.cs:109]: Declare as `private` memeber?

**Review by jameslawesynergy — APPROVED** [review]

File: src/Dfc.CourseDirectory.Core/Validation/ProviderValidation/RuleBuilderExtensions.cs (hunk 1 above)
Comment (gunndabad) [reply src/Dfc.CourseDirectory.Core/Validation/ProviderValidation/RuleBuilderExtensions.cs]: Done

**Pull request merged**